Assets/MixedRealityToolkit.SDK/Experimental/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -655,53 +615,56 @@ private void HandleOneHandMoveUpdated()
             PointerData pointerData = GetFirstPointer();
             IMixedRealityPointer pointer = pointerData.pointer;
 
-            Quaternion targetRotation = Quaternion.identity;
+            var targetPose = new MixedRealityPose(hostTransform.position, hostTransform.rotation);
+            var targetScale = hostTransform.localScale;
```

Comment:
do we need a scale constraint in the one hand move update?

Reply:
Yeah, some scale constraints may affect an object's scale without the user explicitly scaling the object themselves. For example, MaintainApparentSizeConstraint (which I added to demonstrate and test this) scales an object based on distance from the camera.